src/V3Class.cpp

```diff
@@ -46,6 +49,9 @@ class ClassVisitor final : public VNVisitor {
     const AstNodeFTask* m_ftaskp = nullptr;  // Current task
     std::vector<std::pair<AstNode*, AstScope*>> m_toScopeMoves;
     std::vector<std::pair<AstNode*, AstNodeModule*>> m_toPackageMoves;
+    std::unordered_set<AstTypedef*> m_Typedefps;
+    std::unordered_set<AstNodeUOrStructDType*> m_sdtypes;
```

Comment:
```suggestion
    std::set<AstTypedef*> m_Typedefps;
    std::set<AstNodeUOrStructDType*> m_sdtypes;
```

unordered_set has large startup cost - similar below.

Reply:
Fixed